src/components/__tests__/RoutesMapToolbar.cy.js

```diff
@@ -0,0 +1,116 @@
+import RoutesMapToolbar from 'components/routes/RoutesMapToolbar.vue';
+import { i18n } from '../../boot/i18n';
+
+describe('<RoutesMapToolbar>', () => {
+  it('has translation for all strings', () => {
+    cy.testLanguageStringsInContext([], 'index.component', i18n);
+  });
+
+  context('draw disabled', () => {
+    beforeEach(() => {
+      cy.mount(RoutesMapToolbar, {
+        props: {
+          drawEnabled: false,
+        },
+      });
+      cy.viewport('macbook-16');
+    });
+
+    coreTests();
+  });
+
+  context('draw enabled', () => {
+    beforeEach(() => {
+      cy.mount(RoutesMapToolbar, {
+        props: {
+          drawEnabled: true,
+        },
+      });
+      cy.viewport('macbook-16');
+    });
+
+    coreTests();
+    drawEnabledTests();
+  });
+
+  context('draw enabled + delete enabled', () => {
+    beforeEach(() => {
+      cy.mount(RoutesMapToolbar, {
+        props: {
+          drawEnabled: true,
+          deleteEnabled: true,
+        },
+      });
+      cy.viewport('macbook-16');
+    });
+
+    coreTests();
+    drawEnabledTests();
+  });
+});
+
+function coreTests() {
+  it('renders component', () => {
+    cy.dataCy('toolbar-top').should('be.visible');
+    cy.dataCy('toolbar-bottom').should('be.visible');
+  });
+
+  it('renders draw button', () => {
+    cy.dataCy('add-route-button')
+      .should('be.visible')
+      .and('have.css', 'padding', '0px');
+    cy.dataCy('add-route-avatar').should('be.visible');
+    cy.dataCy('add-route-avatar').invoke('height').should('be.equal', 32);
```

Comment:
`const addRouteAvatarIconSize = 32`

Check others repeated icon size number which should be const, please.

Reply:
Refactored selectors and variables to constants.